cmd/company/main.go

```diff
@@ -19,22 +19,34 @@ func main() {
 
 	// "postgresql://user:password@host:port/dbname"
 	connString := `postgresql://appuser:appuser@127.0.0.1:5432/app_db`
-	DB, err := postgres.NewDB(ctx, connString)
+	db, err := postgres.NewDB(ctx, connString)
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer db.Close()
```

Comment:
Здесь сигнатура, возможно, тоже предполагает возврат ошибки. Мы уже и так выходим из функции, так что особо ошибку не обработать... Но, тем не менее, можно её залогировать. Если что-то пойдёт не так, такие логи могут пригодиться. Вот что-то примерно такое (но синтаксис лучше проверить еще раз):


```suggestion
	defer func() {
		err := db.Close()
		if err != nil {
			log.Errorf("closing db conn: %w", err)
		}
	  }
	 }()
```

Reply:
к сожалению, ошибки не возвращает сам пакет, но сам посыл я понял